rts/Rml/SolLua/bind/Context.cpp

```diff
@@ -114,6 +114,57 @@ void bindTable(SolLuaDataModel* data, sol::table& table)
 	}
 }
 
+sol::object getFromTable(sol::table t, std::vector<std::string> keychain, int depth) {
```

Comment:
```suggestion
static sol::object getFromTable(sol::table t, const std::vector<std::string> &keychain, int depth) {
```
I don't know whether `sol::table` is just a handle but perhaps should also be by reference?